Source/WebKit/GPUProcess/mac/com.apple.WebKit.GPUProcess.sb.in

```diff
@@ -845,7 +845,14 @@
 
 ;; Media capture, microphone access
 (with-filter (extension "com.apple.webkit.microphone")
-    (allow device-microphone))
+    (allow device-microphone)
+
+    (allow file-read* (path "/Library/Preferences/com.apple.coreaudio.plist"))
+    (allow user-preference-read (preference-domain "com.apple.coreaudio"))
+    (allow file-read* file-write*
+        (subpath "/Library/Preferences/Audio")
+        (subpath "/tmp/AudioCapture/VP"))
```

Comment:
Would it be sufficient to allow this only on internal builds?

(with-filter (system-attribute apple-internal)
...